java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/SQLRouterSmokeTest.java

```diff
@@ -41,25 +42,38 @@
 public class SQLRouterSmokeTest {
 
     private final Random random = new Random(System.currentTimeMillis());
-    public static SqlExecutor router;
+    public static SqlExecutor clusterRouter;
+    public static SqlExecutor standaloneRouter;
 
     static {
         try {
             SdkOption option = new SdkOption();
             option.setZkPath(TestConfig.ZK_PATH);
             option.setZkCluster(TestConfig.ZK_CLUSTER);
             option.setSessionTimeout(200000);
-            router = new SqlClusterExecutor(option);
-            java.sql.Statement state = router.getStatement();
+            clusterRouter = new SqlClusterExecutor(option);
```

Comment:
must call `Init()` for both `clusterRouter` and `standaloneRouter` before use

Reply:
> must call `Init()` for both `clusterRouter` and `standaloneRouter` before use

which init()? SqlClusterExecutor seems have no `Init()` method

Reply:
`router->Init()` is in `SqlClusterExecutor`.  It's better to change the name.